examples/panel-datalinks/tests/panel.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { PanelEditPage, expect, test } from '@grafana/plugin-e2e';
+import { testIds } from '../src/components/testIds';
+
+test.describe('panel-datalinks panel', () => {
+  let panelEditPage: PanelEditPage;
+  test.beforeEach(async ({ gotoPanelEditPage }) => {
+    panelEditPage = await gotoPanelEditPage({ dashboard: { uid: 'fTh-POZ4k' }, id: '2' });
+  });
+
+  test('should display "No data" when no data is passed to the panel', async ({ page }) => {
+    await expect(page.getByTestId(testIds.panel.noData)).not.toBeVisible();
```

Comment:
Q: should we base the tests on what the user sees instead of the test id as we did before? So maybe check for the No data label instead?

Reply:
yes good shout! will fix